src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1207,6 +1222,41 @@ public void OnDeletePressed()
         editor.RemoveOrganelle(organelleMenu.SelectedOrganelle.Position);
     }
 
+    public override void _Notification(int what)
+    {
+        if (what == NotificationTranslationChanged)
+        {
+            if (autoEvoRunSuccessful.HasValue && autoEvoRunSuccessful.Value == false)
+            {
+                totalPopulationLabel.Text = TranslationServer.Translate("FAILED");
+            }
+
+            if (!string.IsNullOrEmpty(bestPatchName))
+            {
+                bestPatchLabel.Text = string.Format(CultureInfo.CurrentCulture,
```

Comment:
I'd like to see a common method be created where this duplicate code would be placed.